governance/roles.md

```diff
@@ -0,0 +1,259 @@
+as OWNERS of the project directory in this repository to help facilitate and
+ moderate new content to the project.
+
+### Duties of a project lead
+
+Project leads are responsible for the following:
+
+- Adhering to the [project process](process.md)
+- Proposing and refining the project scope, goal and schedule
+- Ensure smooth day-to-day operations of the project. This includes:
+  - Ensuring meetings are scheduled, announced and recorded
+  - Facilitation of meetings, ensuring minutes are recorded
+- Ensure progress and status reports are communicated to the WG
+representative of the project
+- Resolving conflict within the project and escalating to leadership
+when required
+- Outreach and recruitment for the project
+- Provide retrospective of a project to the WG
+
+## Role of WG representatives
+
+Environmental Sustainability WG representatives are leadership executive sponsors of a project. The
+WG representative acts as a point of liaison with the WG leadership team
+and takes on a role of mentorship for the project lead(s).
+
+WG representatives are required to be part of the WG leadership team.
+This includes co-chairs, technical leads and co-chair emeritus roles.
+A WG representative may be of capacity of an individual contributor but
+cannot be the project lead of the same project they are representing.
+
+WG representatives are appointed through the following process:
+
+- A member of the leadership indicates that they would like to be a WG
+representative on a project by:
+  - editing the issue's "WG Representative" field to their github ID.
+  - setting the issue milestone to the corresponding milestone created for
+  their github ID.
+- WG representatives may change based on discretion of co-chairs.
+
+### Duties of a WG representative
+
+WG representatives are responsible for the following:
+
+- Reporting back to the leadership on the status of the project, indicating
+when the project requires external governance help. For example, these
+include:
+  - Project is understaffed or suffers from rampant scope creep.
+  - Project is inactive and needs to be discontinued.
+  - Inter-personal issues within leads and/or members causing disruption.
+- Provide feedback to project lead to whether the current scope and operation
+of a project is in line with the vision and governance of the WG.
+- Mentorship of project leads on matters of leadership and governance process.
+- Provide privilege facilities that a project lead does not have access to.
+Examples are:
+  - Creating a ServiceNow ticket with the CNCF for design help
+  - Uploading of meeting recordings to youtube
+  - Approval of messages to WG mailing list
+
+## TOC liaison
+
+> As WG we search for a deep alignment with the TOC and community, therefore, we apply TAG rules/processes
+
+The [CNCF TAG](https://github.com/cncf/toc/blob/master/tags) process identifies
+a TOC Liaison.  The WG Chairs are responsible for establishing effective
```

Comment:
No, the TOC Liason is a member of TOC who works with you all to help prioritize. Just strike this section. :)